README.md

```diff
@@ -43,10 +43,11 @@ Application Options:
   -s, --all-servers   Use parallel queries to speed up resolving by querying all upstream servers simultaneously
   -d, --ipv6-disabled Disable IPv6. All AAAA requests will be replied with No Error response code and empty answer 
       --edns          Use EDNS Client Subnet extension
+      --edns-addr=    Send EDNS Client Address
+      --version       Prints the program version
```

Comment:
What's wrong with the --version being in `Help Options` ?
This is not what we need in this patch for sure.